src/main/java/teammates/ui/controller/InstructorCourseInstructorAddAction.java

```diff
@@ -85,8 +94,22 @@ private InstructorAttributes extractCompleteInstructor(String courseId, String i
         return instructorToAdd;
     }
     
-    private InstructorAttributes updateBasicInstructorAttributes(String courseId, String instructorName, String instructorEmail,
-            String instructorRole, boolean isDisplayedToStudents, String displayedName) {
+    /**
+     * Creates a new instructor with basic information.
+     * This consists of everything apart from custom privileges.
+     * 
+     * @param courseId              Id of the course the instructor is being added to.
+     * @param instructorName        Name of the instructor.
+     * @param instructorEmail       Email of the instructor.
+     * @param instructorRole        Role of the instructor.
+     * @param isDisplayedToStudents Whether the instructor should be visible to students.
+     * @param displayedName         Name to be visible to students.
+     *                                  Should not be {@code null} even if {@code isDisplayedToStudents} is false.
+     * @return An instructor with basic info, excluding custom privileges
+     */
+    private InstructorAttributes updateBasicInstructorAttributes(String courseId, String instructorName,
```

Comment:
The method name `update` doesn't reflect that a creation is taking place. Change the method name to something like `createInstructorWithBasicAttributes` or alike.